client/data/disputes/hooks.ts

```diff
@@ -32,33 +42,111 @@ export const useDispute = (
 	return { dispute, isLoading, doAccept };
 };
 
-export const useDisputeEvidence = () => {
+export const useDisputeEvidence = (): {
+	updateDispute: ( data: Dispute ) => void;
+} => {
 	const { updateDispute } = useDispatch( STORE_NAME );
 	return { updateDispute };
 };
 
 export const useDisputes = ( {
 	paged,
 	per_page: perPage,
-}: {
-	paged: string;
-	per_page: string;
-} ) =>
+	store_currency_is: storeCurrencyIs,
+	match,
+	date_before: dateBefore,
+	date_after: dateAfter,
+	date_between: dateBetween,
+	status_is: statusIs,
+	status_is_not: statusIsNot,
```

Comment:
The filter names haven't been updated as per the [comment made on the issue](https://github.com/Automattic/woocommerce-payments/issues/3591#issuecomment-1005718727). currency and date filter will not be taken into account by server (also nor will throw any errors or complains).

This comment goes for many use places thoughout the changes

Reply:
As we agreed on p1642755254002400-slack-C02AW097TFE we would keep the consistency between models and map the properties on fetch.